core-primitives/sgx/crypto/Cargo.toml

```diff
@@ -38,8 +38,10 @@ std = [
     "sp-core/std",
     "serde/std",
     "serde_json/std",
+    "sgx-crypto-helper/default",
 ]
 sgx = [
+    "sgx-crypto-helper/mesalock_sgx",
```

Comment:
Yes, I believe that this is because some crates need to import `sgx-crypto-helper`, but fail to do so. If things are executed from root level, it works because the `sgx-crypto-helper` is leaked into the crate from another crate, but some crates fail to compile without this leak.

Reply:
Ah wait, maybe you even fixed this in the correct place. There is hope :)

Reply:
That was my intention at least 😄